clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/core/CloudProvider.java

```diff
@@ -1,49 +1,51 @@
 /*
- * Copyright 2015 Netflix, Inc.
+ * Copyright 2020 Armory, Inc.
```

Comment:
Mind the copyright header changes.

Reply:
Yeah, I think that happened automatically. I wasn't sure what the protocol is for keeping/changing headers when a file is rewritten. I'm definitely not trying to take credit for existing work.